readme.rst

```diff
@@ -102,17 +102,53 @@ This will do several things:
 The ``run`` target is therefore very convenient when preparing to submit a
 change to the documentation.
 
-Submit your change
-~~~~~~~~~~~~~~~~~~
+Local Checks
+~~~~~~~~~~~~
+
+Before committing and pushing changes, it's a good practice to run various checks locally to catch issues early in the development process.
 
-Prior to submitting your change, it is recommended to do a fresh build in order
-to surface any errors that may cause build issues on the RTD side:
+Local build
+^^^^^^^^^^^
+
+Build the docs as HTML to surface any build errors that would occur in RTD:
 
 .. code-block:: none
 
    make clean-doc
    make html
 
+Spelling Check
+^^^^^^^^^^^^^^
+
+Ensure there are no spelling errors in the documentation:
+
+.. code-block:: shell
+
+   make spelling
+
+Inclusive Language Check
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure the documentation uses inclusive language:
+
+.. code-block:: shell
+
+   make woke
+
+Link Check
+^^^^^^^^^^
+
+Validate links within the documentation:
+
+  .. code-block:: shell
```

Comment:
Wrong indentation (always check the preview ;) )

Reply:
Ah good catch (and advice :) )